### ISSUE TITLE
PHP 7 support

### ISSUE THREAD

**Issue by akroii** [codefog/contao-news_categories#94]
Can you please provide PHP7 support?
```
Warning: Declaration of NewsCategories\ModuleNewsMenu::compileWeeks($arrData, $strUrl) should be compatible with Contao\ModuleNewsMenu::compileWeeks($arrData) in system/modules/news_categories/modules/ModuleNewsMenu.php on line 21
#0 system/modules/news_categories/modules/ModuleNewsMenu.php(21): __error(2, 'Declaration of ...', '/homepages/10/d...', 21, Array)
#1 system/modules/core/library/Contao/ClassLoader.php(161): include('/homepages/10/d...')
#2 [internal function]: Contao\ClassLoader::load('ModuleNewsMenu')
#3 [internal function]: spl_autoload_call('ModuleNewsMenu')
#4 system/modules/core/elements/ContentModule.php(43): class_exists('ModuleNewsMenu')
#5 system/modules/core/library/Contao/Controller.php(484): Contao\ContentModule->generate()
#6 system/modules/core/modules/ModuleArticle.php(213): Contao\Controller::getContentElement(Object(Contao\ContentModel), 'left')
#7 system/modules/core/modules/Module.php(287): Contao\ModuleArticle->compile()
#8 system/modules/core/modules/ModuleArticle.php(67): Contao\Module->generate()
#9 system/modules/core/library/Contao/Controller.php(417): Contao\ModuleArticle->generate(false)
#10 system/modules/core/library/Contao/Controller.php(277): Contao\Controller::getArticle(Object(Contao\ArticleModel), false, false, 'left')
#11 system/modules/core/pages/PageRegular.php(133): Contao\Controller::getFrontendModule('0', 'left')
#12 system/modules/core/controllers/FrontendIndex.php(285): Contao\PageRegular->generate(Object(Contao\PageModel), true)
#13 index.php(20): Contao\FrontendIndex->run()
#14 {main}
```

Contao 3.5.17
News Categories 2.7.0 Build 3

**Comment by fritzmg** [comment]
This has nothing to do with PHP 7. This is an error that was fixed in version `2.7.3`, see #92.

**Comment by qzminski** [comment]
Thanks for sorting it out @fritzmg !
